test/CMakeLists.txt

```diff
@@ -14,10 +14,10 @@ set(TPP_OPT_TEST_DEPENDS
         tpp-run
         )
 
-add_lit_testsuite(check "Running the regression tests"
+add_lit_testsuite(tpp-check "Running the regression tests"
```

Comment:
We used to have formatting checks into `check-all` but now we don't anymore, so `check` can become `check-tpp`. We don't need a check-all any more.

Please, just remove the last line in the main CMake file:
```
# Check-all is just check, for now
add_custom_target(check-all DEPENDS check)
```

and rename this target to `check-tpp` instead. Then change the build scripts to just call `check-tpp`.

Reply:
done